pkg/crc/podman/podman.go

```diff
@@ -0,0 +1,72 @@
+package podman
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	crcos "github.com/code-ready/crc/pkg/os"
+)
+
+const (
+	rootlessConn = "crc"
+	rootfulConn  = "crc-root"
+)
+
+var podmanExecutablePath = filepath.Join(constants.CrcOcBinDir, constants.PodmanRemoteExecutableName)
+
+func run(args ...string) (string, string, error) {
+	return crcos.RunWithDefaultLocale(podmanExecutablePath, args...)
+}
+
+func addSystemConnection(defaultCon bool, name, identity, uri string) error {
+	// podman system connection add --identity /Users/prkumar/.ssh/podman-machine-default crc ssh://core@localhost:2222/run/user/1000/podman/podman.sock
+	args := []string{"system", "connection", "add", "--identity", identity, name, uri}
+	if defaultCon {
+		args = append(args, "-d")
+	}
+	if _, stderr, err := run(args...); err != nil {
+		return fmt.Errorf("failed to add podman system connection %v: %s", err, stderr)
+	}
+	return nil
+}
+
+// AddRootlessSystemConnection and makes it default
```

Comment:
A part of the  sentence seems to be missing. Not sure `AddRootlessSystemConnection` and `AddRootfulSystemConnection` should behave differently wrt to default. There are `MakexxxSystemConnectionDefault`  methods already, `addPodmanSystemConnections` could just call these, this way it is very clear what the default is, the API is symmetrical, ...

Reply:
@cfergeau I was doing 2 separate thing with single function :(